content/en/workflows/setup.md

```diff
@@ -1,9 +1,82 @@
 ---
-title: Set up Workflows
+title: Authentication
 kind: documentation
 disable_toc: false
 further_reading:
-- link: "logs/processing/pipelines"
+- link: "/integrations/"
   tag: "Documentation"
-  text: "Log processing pipelines"
----
\ No newline at end of file
+  text: "Learn about integrations"
+---
+
+<div class="alert alert-warning">Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.</div>
+
+Because workflow actions connect with external software systems, you may need to authenticate your Datadog account to the corresponding integration or integrations. A workflow can run successfully only if every workflow action that requires authentication can verify the identity of your Datadog account.
+
+Integration actions can be authenticated in two ways:
+- Credentials and permissions configured in the integration tile
+- Custom connection credentials
+
+## Integration tile credentials
+
+Credentials and account authentication that you set up in Datadog integration tiles automatically propagate to the corresponding actions in Workflows. Configure the integration tiles by following instructions in [Datadog Integrations][1].
+
+## Custom connection credentials
+
+You can create custom credentials within Workflows. These custom connection credentials are only available for use within the Workflows product.
+
+### Create custom connection
+
+1. In the left navigation, go to **Integrations** -> **Workflows** to access the [Workflows list][2].
+1. Click **Connections** in the upper right.
+1. Select **New Connection**. A dialog box appears.
+1. Choose an integration schema. The **Custom** option allows you to authenticate using a token or username/password combination. The other options use authentication schemas associated with the integration.
+1. Fill in the remaining information required by the integration schema.
+1. Click **Create** to save your custom connection.
+
+### Restrict connection use
+
+You can set permissions on each connection to limit modifications or restrict their use. The granular permissions include **Editor** and **Resolver**.
+
+1. Navigate to the [Workflows list][2].
+1. Click **Connections** in the upper right. A list of connections appears.
+1. Hover over the connection on which you would like to set granular permissions. Edit, permissions, and delete icons appear on the right.
+1. Click the lock (permissions) icon.
```

Comment:
```suggestion
1. Click the padlock (**Permissions**) icon.
```
A bit of a nit, but I prefer padlock because the noun form is more common, hopefully leading to less confusion for non-native English readers.